builtin/logical/aws/secret_access_keys.go

```diff
@@ -246,6 +280,18 @@ func (b *backend) secretAccessKeysCreate(
 		}
 	}
 
+	for _, group := range role.IAMGroups {
+		// Add user to IAM groups
+		_, err = iamClient.AddUserToGroup(&iam.AddUserToGroupInput{
+			UserName:  aws.String(username),
+			GroupName: aws.String(group),
+		})
+		if err != nil {
+			return logical.ErrorResponse(fmt.Sprintf(
```

Comment:
nit: the `fmt.Sprintf` isn't needed. `logical.ErrorResponse` can take a format string and args.